apps/frontend/src/libs/components/image/image.tsx

```diff
@@ -0,0 +1,29 @@
+import styles from "./styles.module.css";
+
+type Properties = {
+	alt: string;
+	height?: string;
+	isCircular?: boolean;
```

Comment:
```suggestion
	shape?: 'default' | 'circle';
```
default is `default`